workers/unity/Assets/Playground/Scripts/Player/VirtualJoystick.cs

```diff
@@ -0,0 +1,49 @@
+﻿using UnityEngine;
+using UnityEngine.EventSystems;
+using UnityEngine.UI;
+
+public class VirtualJoystick : MonoBehaviour, IDragHandler, IPointerUpHandler, IPointerDownHandler
+{
+    private Image bgImg;
```

Comment:
If you tag there with `[SerializeField]` then the references can be set through the inspector, instead of using code in `Start`